guidelines/terms/22/perimeter.html

```diff
@@ -1,7 +1,6 @@
 <dt class="new"><dfn>Perimeter</dfn></dt>
 <dd class="new">
 	<p class="change">New</p>	
-   <p>the continuous line forming the boundary of a shape. The perimeter calculation for a rectangle is 2<em>h</em>+2<em>w</em>, where <em>h</em> is the height and <em>w</em> is the width. The perimeter of a circle is 2𝜋<em>r</em>.
-    </p>
+   <p>the continuous line forming the boundary of a shape. The perimeter calculation for a rectangle is 2<em>h</em>+2<em>w</em>, where <em>h</em> is the height and <em>w</em> is the width. The perimeter of a circle is 2𝜋<em>r</em>.</p>
```

Comment:
to be consistent with other definitions, suggest:
<p>continuous line forming the boundary of a shape</p>
<p>The perimeter calculation for a rectangle is 2<em>h</em>+2<em>w</em>, where <em>h</em> is the height and <em>w</em> is the width. The perimeter of a circle is 2𝜋<em>r</em>.</p>